frontend/package.json

```diff
@@ -19,6 +19,7 @@
     "@types/react-table": "^7.7.9",
     "axios": "^0.24.0",
     "formik": "^2.2.9",
+    "jsdoc": "^3.6.11",
```

Comment:
Not at all opposed to using jsdoc, but looking forward to discussing the implementation (and commenting in general) as a team during code talk. :)